libs/designsystem/src/lib/components/item/item.component.ts

```diff
@@ -14,6 +14,9 @@ export class ItemComponent {
   selectable: boolean;
   @Input()
   reorderable: boolean;
+  @HostBinding('class')
+  @Input()
+  size: 'xs' | 'sm' | 'md' = 'md';
```

Comment:
Please align with other component and declare exported enum type for ItemSize

Reply:
I created an issue for this (#1408), so if ok with you, I'll leave it until then to align this for all the comps.

Reply:
Scratch that, you were right. Implemented in f925b2b2051a13007f3b25ffe21670de7a79e2c2